snuba/migrations/table_engines.py

```diff
@@ -49,8 +59,11 @@ def get_sql(self) -> str:
 
         return sql
 
-    def _get_engine_type(self) -> str:
-        return "MergeTree()"
+    def _get_engine_type(self, single_node: bool, table_name: str) -> str:
+        if single_node:
+            return "MergeTree()"
+        else:
+            return f"ReplicatedMergeTree('/clickhouse/tables/{{layer}}-{{shard}})/{table_name}', '{{replica}}')"
```

Comment:
Do you expect we will ever support more parameters other than the engine name to be specific of the cluster mode? Like the SETTINGS ? I think we should keep that in mind going on and have a strategy to modify this framework accordingly.

Reply:
I don't think we are ever applying different settings for replicated tables. But I think we might need something a bit different for the distributed and materialized view engines. Will try to tackle this in a follow up.